docs/boshdirector.adoc

```diff
@@ -1,20 +1,32 @@
-= Installing Bosh Director
+= Configuring Ops Manager Director
```

Comment:
i recall the pcf immersion instructions also called this configuration the ops manager director.  but honestly what you're really doing is installing bosh.  and i think that's important to point out.